src/docs/OrchardCore.Docs.csproj

```diff
@@ -7,4 +7,14 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="reference\modules\Spatial\README.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="reference\modules\Spatial\README.md">
```

Comment:
Needed or just VS adding?

Reply:
You are right I need to check this and remove it. It's VS adding.